docs(deploy): add some note for databend-hdfs deploy (#147)
datafuselabs/databend-docs: docs/doc/10-deploy/02-deploying-databend.md

</TabItem>

<TabItem value="HDFS" label="HDFS">

> Before deploying Databend, make sure you have successfully set up your Hadoop environment, and the following tasks have been completed:

Reviewer: ```suggestion
Before deploying Databend, make sure you have successfully set up your Hadoop environment, and completed the following tasks:
```

> - Your system already has an Java SDK installed with JVM support.

Reviewer: ```suggestion
- Your system already has a Java SDK installed with JVM support.
```

- Get the name node URL for connecting to HDFS.

> - You have already downloaded the Hadoop release in your system and can access the JAR packages within it.

Reviewer: ```suggestion
- You have already downloaded the Hadoop release to your system, and you can access the JAR packages in the  release.
```

b. Download and extract the latest Databend release for your platform from [GitHub Release](https://github.com/datafuselabs/databend/releases):

:::note

> If you need to use HDFS as the storage backend for Stage or Databend itself, please choose a release with "hdfs" in its name, for example:

Reviewer: ```suggestion
To use HDFS as the storage backend, download a release with a file name formatted as `databend-hdfs-${version}-${target-platform}.tar.gz`.
```

> ```bash

Reviewer: remove this code snippet

e. Run the following command to start the Query node:

:::note

> If you need to use HDFS as the storage backend for Stage or Databend itself, please set the following environment variables correctly to ensure proper invocation:

Reviewer: ```suggestion
When using HDFS as the storage backend, ensure to set the following environment variables:
```

```bash
export JAVA_HOME=/path/to/java
export LD_LIBRARY_PATH=${JAVA_HOME}/lib/server:${LD_LIBRARY_PATH}
export HADOOP_HOME=/path/to/hadoop

> export CLASSPATH=/all/hadoop/jar/files

Author: for example:

```
export LD_LIBRARY_PATH={$JAVA_HOME}/lib/server/
export JAVA_HOME=/usr/lib/jvm/java-21-jdk
export HADOOP_HOME={$HOME}/hadoop-3.3.6
export CLASSPATH=$(find $HADOOP_HOME -iname "*.jar" | xargs echo | tr ' ' ':')
```

Reviewer: It would be better if adding this example below the environment variables list.
